test/index.js

```diff
@@ -88,3 +88,22 @@ test('only permitted fields should be used', function (t) {
 
   t.end()
 })
+
+test('symbols should not overlap', function (t) {
+  const symbols = Object.keys(contractMap).map(contract => contractMap[contract].symbol)
+  const symbolsCheck = Object.create(null)
```

Comment:
Better to use a `Map` or `Set` than an object. Will need to change the `in` operator to match